3.3/test/django-test-apps/no-static/run.py

```diff
@@ -0,0 +1,2 @@
+import os
+os.execl('./manage.py', './manage.py', 'runserver', '0.0.0.0:8080')
```

Comment:
For your local dev, to speed up the process you can do whatever you feel comfortable with, but in the repo I'd prefer to avoid such things. The additional 2 mins for running tests are not a problem, and we should evangelize users to run applications inside of OpenShift correctly, rather than going the easy path :wink: